MikuMikuModel/Nodes/Sprites/SpriteNode.cs

```diff
@@ -79,6 +85,39 @@ private void CalculateNdcValues()
 
         protected override void Initialize()
         {
+            RegisterExportHandler<Bitmap>(filePath =>
+            {
+                var imageFormat = ImageFormat.Png;
+
+                if (!string.IsNullOrEmpty(filePath))
+                {
+                    string extension = Path.GetExtension(filePath).Trim('.').ToLowerInvariant();
+                    switch (extension)
+                    {
+                        case "png":
+                            imageFormat = ImageFormat.Png;
+                            break;
+
+                        case "jpg":
+                        case "jpeg":
+                            imageFormat = ImageFormat.Jpeg;
+                            break;
+
+                        case "gif":
+                            imageFormat = ImageFormat.Gif;
+                            break;
+
+                        case "bmp":
+                            imageFormat = ImageFormat.Bmp;
+                            break;
+
+                        default:
+                            throw new ArgumentException("Image format could not be detected", nameof(filePath));
+                    }
+                }
+
+                SpriteCropper.Crop(Data, FindParent<SpriteSetNode>().Data).Save(filePath, imageFormat);
```

Comment:
Wrap the returning Bitmap object in a using statement to dispose of it immediately.